### DISCUSSION TITLE
Dependency-Track stuck on login page

### DISCUSSION THREAD

**Discussion by Poremich** [DependencyTrack/dependency-track#3534] [Q&A]
After installing Dependency-Track on Kali Linux I go to the localhost:8080 and try to log in using the default credentials admin admin, but when I click Login nothing happens and I'm basically stuck on login page: 
![image](https://github.com/DependencyTrack/dependency-track/assets/94555304/421c9e57-6b49-4589-b9bb-d1fd445dc6fc)

**Comment by AFulgens** [comment]
See https://github.com/DependencyTrack/dependency-track/issues/1074#issuecomment-2315131762
